pom.xml

```diff
@@ -563,7 +563,7 @@
         <version.org.keycloak.keycloak-saml-wildfly-subsystem>18.0.2</version.org.keycloak.keycloak-saml-wildfly-subsystem>
         <version.org.kohsuke.metainf-services>1.11</version.org.kohsuke.metainf-services>
         <version.org.lz4.lz4-java>1.8.0</version.org.lz4.lz4-java>
-        <version.org.opensaml.opensaml>4.2.0</version.org.opensaml.opensaml>
+        <version.org.opensaml.opensaml>4.3.0</version.org.opensaml.opensaml>
```

Comment:
Is there a reason not to use 4.3.3?

Reply:
4.3.0 is the version which is used in CXF 4.0.5.  4.3.3 is not tested in wss4j and cxf.